translations/messages.ru.xlf

```diff
@@ -266,7 +266,7 @@
             </trans-unit>
             <trans-unit id="post.num_comments">
                 <source>post.num_comments</source>
-                <target>%count% комментарий|%count% комментария|%count% комментариев</target>
+                <target>{count, plural, one {# комментарий} many {# комментария} other {# комментариев}}</target>
```

Comment:
The 3 Russian variants are `one`, `few` and `many`: http://www.unicode.org/cldr/charts/latest/supplemental/language_plural_rules.html#ru-comp

(and yes, using `other` would have been better than `many`, but they cannot fix it anymore for BC reasons: http://cldr.unicode.org/index/cldr-spec/plural-rules#TOC-Choosing-Plural-Category-Names)